soundcard/pulseaudio.py

```diff
@@ -11,7 +11,7 @@
 import collections
 import time
 import re
-import numpy
+import numpy as np
```

Comment:
Personally, I am not a fan of this for libraries. Libraries should strive for readability first, and this mostly makes it easier to *write*, though at a cost in ease of reading.

Reply:
I saw it in coreaudio.py, so I thought I could do it. 
You obviously take the decision on this, it's your repo. But for me it doesn't come at a cost for ease of reading, I'm too used to it.